Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -9,7 +9,30 @@ public static class ArrayOperations
         /// <param name="array">Input array in a random order.</param>
         public static void Sort(int[] array)
         {
-            // ToDo: implement.
+            if (array == null || array.Length == 0)
+            {
+                return;
+            }
+            else
+            {
+                for (int i = 0; i < array.Length -1; i++)
+                {
+                    for (int j = i + 1; j < array.Length; j++)
+                    {
+                        if (array[i] > array[j])
+                        {
+                            Swap(array, i, j);
+                        }
+                    }
+                }
+            }
+        }
+
+        private static void Swap(int[] array, int i, int j)

```

Comment:
Well done by moving this to a step function!

Reply:
It should be placed at the very bottom, after all the public functions.
private functions are a part of public functions- they are a detail and if we care about this detail, we should not be forced to know about it all the time. If we need to know- we will scroll. Otherwise, it should be not in front.